src/net_processing.cpp

```diff
@@ -146,6 +147,65 @@ static constexpr uint32_t MAX_GETCFILTERS_SIZE = 1000;
 static constexpr uint32_t MAX_GETCFHEADERS_SIZE = 2000;
 /** the maximum percentage of addresses from our addrman to return in response to a getaddr message. */
 static constexpr size_t MAX_PCT_ADDR_TO_SEND = 23;
+/** Static component of the salt used to compute short txids for transaction reconciliation. */
+static const std::string RECON_STATIC_SALT = "Tx Relay Salting";
+/** Used to convert a floating point reconciliation coefficient q to an int for transmission. */
+static constexpr double Q_PRECISION{2 << 12};
```

Comment:
BIP comment: why 12 bits? You could have 16 bits of accuracy with the 2 bytes that it takes.

Reply:
q is computed as `(total_missing - abs(local_missing - remote_missing)) / min_size`

So the bounds on q are [0...2] I think?
if abs = 0, total_missing = 2 * min_size is an upper bound.
If abs !=0, well. Ideally, we need to solve this equation properly :)

So it should be (2 << 14) - 1 I think assuming [0..2] for now.